lib/recordingSession.dart

```diff
@@ -16,12 +17,16 @@ class RecordingPage extends StatefulWidget {
 }
 
 class _RecordingPageState extends State<RecordingPage> {
+  PushNotificationsManager pm;
```

Comment:
rename variable to something more readible like `pushNotificationsManager` or `notificationsManager`